sympy/stats/stochastic_process_types.py

```diff
@@ -2,15 +2,16 @@
 import random
 
 import itertools
-from typing import Sequence as tSequence, Union as tUnion, List as tList, Tuple as tTuple
+from typing import (Sequence as tSequence, Union as tUnion, List as tList,
+        Tuple as tTuple, Set as tSet)
```

Comment:
This naming convention isn't the greatest (not that I can think of a better one). Maybe we should require all typing imports to come from `sympy.typing` which could provide common alias names. Even the fact that some files import Tuple or Dict as is because they don't use the SymPy Tuple or Dict is confusing.

Reply:
Given the ubiquity of typing now in common editors and so on I wonder if sympy needs to bite the bullet and start calling them `symTuple` or `BasicTuple` or something so that `Tuple` can just be for typing. Or actually maybe there was a PEP that was going to change it so you could just do `tuple[int]` etc...

Reply:
I'm also adding type hint, and I expect an official naming convention to be documented somewhere.